.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
+
 php:
-    - 5.5
-    - 5.4
+    - 7.2
```

Comment:
It's completely unsupported in versions below 7.2?

I think it needs some little mention in the README in that case, like "For old PHP versions, use version `0.4.0`".

I'll tag this release `1.0.0` or something. Ironic, since I haven't fully tested it heh.

Reply:
yes. Minimal requirements for guzzle 7 is php 7.2.5 (http://docs.guzzlephp.org/en/stable/overview.html#requirements)

Reply:
> I'll tag this release `1.0.0` or something. Ironic, since I haven't fully tested it heh.

All your tests are passed :)  In my scenario also all work fine. I hope, it is not broken :)